cloud-formation/scripts/post-dev-stack-creation.sh

```diff
@@ -3,6 +3,8 @@
 source ./utils.sh
 source ./stack-name.sh
 
+export AWS_PROFILE=media-service
```

Comment:
Note that a shell script cannot set an environment variable outside of its execution. IOW `AWS_PROFILE` is only set within that script, unless you `source` the script instead of executing it.